src/plugins/data_explorer/public/components/app_container.tsx

```diff
@@ -13,25 +13,46 @@ import { View } from '../services/view_service/view';
 import './app_container.scss';
 
 export const AppContainer = ({ view, params }: { view?: View; params: AppMountParameters }) => {
+  const isMobile = useIsWithinBreakpoints(['xs', 's', 'm']);
   // TODO: Make this more robust.
   if (!view) {
     return <NoView />;
   }
 
   const { Canvas, Panel, Context } = view;
 
+  const MemoizedPanel = memo(Panel);
+  const MemoizedCanvas = memo(Canvas);
+
   // Render the application DOM.
   return (
     <EuiPage className="deLayout" paddingSize="none">
       {/* TODO: improve fallback state */}
       <Suspense fallback={<div>Loading...</div>}>
         <Context {...params}>
-          <Sidebar>
-            <Panel {...params} />
-          </Sidebar>
-          <EuiPageBody className="deLayout__canvas">
-            <Canvas {...params} />
-          </EuiPageBody>
+          <EuiResizableContainer direction={isMobile ? 'vertical' : 'horizontal'}>
+            {(EuiResizablePanel, EuiResizableButton) => (
+              <>
+                <EuiResizablePanel
+                  initialSize={25}
+                  minSize="260px"
+                  mode="collapsible"
+                  style={isMobile ? { paddingBottom: 8 } : { paddingRight: 8 }}
```

Comment:
```suggestion
                  paddingSize="none"
                  style={isMobile ? { paddingBottom: 8 } : { paddingRight: 8 }}
```

You need both styles to get rid of the unnecessary padding here.